### ISSUE TITLE
There are problems when compiling to JS

### ISSUE THREAD

**Issue by 364734461** [paljs/create-nexus-type#17]
RUN `cnt -s -mq -m -q -f -o --js`

generate ok. But generated code Error.

```shell
export const User = objectType({
^^^^^^

SyntaxError: Unexpected token 'export'
...
```

Possible reasons ：

```js
    fileContent += `export const ${model.name} = objectType({
...
```

**Comment by 364734461** [comment]
fix

```js
fileContent += `${args['--js'] ? '' : 'export '}const ${
      model.name
    }
```

**Comment by AhmedElywa** [comment]
Sorry for the delay I just pushed a new release with the fix `1.2.6 `
